Lib/fontTools/ttLib/tables/O_S_2f_2.py

```diff
@@ -299,6 +300,19 @@ def recalcUnicodeRanges(self, ttFont, pruneOnly=False):
 		self.setUnicodeRanges(bits)
 		return bits
 
+	def recalcXAvgCharWidth(self, ttFont):
```

Comment:
I’d drop the `X` from the function name.

Reply:
dropped